src/ralph/ssl_certificates/models.py

```diff
@@ -27,10 +27,22 @@ class SSLCertificate(AdminAbsoluteUrlMixin, BaseObject):
         help_text=_('Full certificate name'),
         max_length=255
     )
+    domain_ssl = models.CharField(
+        verbose_name=_('domain name'),
+        blank=True,
+        help_text=_('Full domain name'),
+        max_length=255
+    )
     certificate_type = models.PositiveIntegerField(
         choices=CertificateType(),
         default=CertificateType.ov.id,
     )
+    service_environment = models.ForeignKey(
```

Comment:
`service_env` field is added by default through `BaseObject`

Reply:
Of course, I'll change it.